tests/test_s3_to_json_s3.py

```diff
@@ -0,0 +1,477 @@
+                  "format" : "date-time"
+                },
+                "contentType" : {
+                  "type" : "string",
+                  "description" : "The content type of the file."
+                },
+                "identifier" : {
+                  "type" : "string",
+                  "description" : "The identifier for the result."
+                },
+                "stepPath" : {
+                  "type" : "string",
+                  "description" : "The full path to the result if it is within the step history."
+                },
+                "jsonSchema" : {
+                  "type" : "string",
+                  "description" : "The uri for the json schema if the content type is 'application/json'.",
+                  "format" : "uri"
+                },
+                "metadata" : {
+                  "description" : "Any additional metadata about this file."
+                }
+              },
+              "required" : [
+                "filename",
+                "timestamp"
+              ],
+              "additionalProperties" : False,
+              "examples" : [
+                {
+                  "filename" : "foo.json",
+                  "timestamp" : "2022-06-14T11:29:59.915-07:00",
+                  "contentType" : "application/json",
+                  "identifier" : "foo",
+                  "stepPath" : "Bar/foo",
+                  "jsonSchema" : "http://example.org/schemas/v1/Foo.json",
+                  "metadata" : {
+                    "value" : 1
+                  }
+                }
+              ]
+            }
+          },
+          "properties" : {
+            "appName" : {
+              "type" : "string",
+              "description" : "Name of the app that built the archive."
+            },
+            "appVersion" : {
+              "type" : "string",
+              "description" : "Version of the app that built the archive."
+            },
+            "deviceInfo" : {
+              "type" : "string",
+              "description" : "Information about the specific device."
+            },
+            "deviceTypeIdentifier" : {
+              "type" : "string",
+              "description" : "Specific model identifier of the device."
+            },
+            "files" : {
+              "type" : "array",
+              "description" : "A list of the files included in this archive.",
+              "items" : {
+                "$ref" : "#/definitions/FileInfo"
+              }
+            }
+          },
+          "required" : [
+            "appName",
+            "appVersion",
+            "deviceInfo",
+            "deviceTypeIdentifier",
+            "files"
+          ]
+        }
+        return metadata_json_schema
+
+    def test_get_dataset_mapping(self, dataset_mapping_uri):
+        data_mapping = get_data_mapping(data_mapping_uri=dataset_mapping_uri)
+        assert isinstance(data_mapping, dict)
```

Comment:
Nit: Think about if you wanted to assert equals to a dictionary.

Reply:
I created a fixture for both `dataset_mapping.json` and `schema_mapping.json` that reads the mappings from a local path and does a direct comparison with the dict fetched here.